src/shogun/classifier/svm/LibSVM.cpp

```diff
@@ -61,7 +74,7 @@ bool CLibSVM::train_machine(CFeatures* data)
 	if (m_linear_term.vlen>0)
 	{
 		if (m_labels->get_num_labels()!=m_linear_term.vlen)
-            SG_ERROR("Number of training vectors does not match length of linear term\n")
```

Comment:
technically, whitespace changes should be in separate patches. But dont worry for now, just for future prs